bioptim/limits/constraints.py

```diff
@@ -94,6 +94,15 @@ def print(self):
         # TODO: Print all elements in the console
         raise NotImplementedError("Printing of ConstraintList is not ready yet")
 
+    @staticmethod
+    def get_nlp_constraints(nlp):
```

Comment:
Why not send ocp instead? So like this you can get the ocp.g too?

Reply:
Good idea, I did so in the last commit.

Reply:
Done